### ISSUE TITLE
QTI import tool

### ISSUE THREAD

**Issue by jonatanschroeder** [PrairieLearn/PrairieLearn#4074]
QTI is used as a format to export assessments and question banks. Importing these banks would make it easier for instructors to migrate from other platforms or to use public question banks like Canterbury.

**Comment by craig-willis** [comment]
I'm exploring porting questions from Coursera to PL and came across this issue. Coursera supports QTI as an export format. I may have a student with cycles to work on some sort of QTI migration tool and wondered if there had been any further thought on this.

**Comment by nwalters512** [comment]
There's been no further work on this. We'd definitely like to see what you and your student can come up with! That could potentially be added to PL in the `tools/` directory, or we could link to it as an externally-developed option.

**Comment by reteps** [comment]
We are looking at this again.

**Comment by SybelBlue** [comment]
Added sub-issue for Learnvia specific tracking.
*(Sorry for the close, mis-clicked comment button 😬)*

**Comment by jonatanschroeder** [comment]
Can this be considered resolved?

**Comment by reteps** [comment]
There is some follow up work we are doing around question bank support. Once the feature flag is flipped, I think we can.

**Comment by jonatanschroeder** [comment]
Since this is no longer behind a feature flag can it be considered resolved?

**Comment by nwalters512** [comment]
Yep! Closing. For future reference, the docs for this new functionality are here: https://docs.prairielearn.com/importingContent/
